packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -86,6 +86,7 @@ const PostTypeToTag: Record<PostType, FunctionComponent> = {
   [PostType.Share]: SharePostCard,
   [PostType.Welcome]: WelcomePostCard,
   [PostType.Freeform]: WelcomePostCard,
+  [PostType.VideoYouTube]: ArticlePostCard,
```

Comment:
Reusing this at the moment since it doesn't require much changes, let me know if this is not the best approach for this moment.